composer.json

```diff
@@ -38,7 +38,9 @@
         "webklex/composer-info": "^0.0.1",
         "composer" : "^2.0.0",
         "zbateson/mail-mime-parser": "^2.4",
-        "league/commonmark": "^2.4"
+        "league/commonmark": "^2.4",
+        "twbs/bootstrap": "5.0.2",
```

Comment:
5.3.2 is most recent release
^ or ~ should be used to automate

Reply:
Upgraded. Thanks